package.json

```diff
@@ -9,23 +9,23 @@
   "private": true,
   "repository": {
     "type": "git",
-    "url": "https://github.com/bsides/horizoverlay.git"
+    "url": "https://github.com/Rawrington/horizoverlay.git"
   },
   "bugs": {
-    "url": "https://github.com/bsides/horizoverlay/issues"
+    "url": "https://github.com/Rawrington/horizoverlay/issues"
   },
-  "homepage": "https://bsides.github.io/horizoverlay",
+  "homepage": "https://rawrington.github.io/horizoverlay",
```

Comment:
The reference from the original repo should still be the same. Could you make this change?

Reply:
yeah I'll sort it

Reply:
oh it was sorted